CRM/Activity/Form/Search.php

```diff
@@ -98,6 +98,13 @@ public function preProcess() {
     }
     else {
       $this->_formValues = $this->get('formValues');
+
+      if ($this->_force) {
+        // If we force the search then merge form values with the defaults
+        // (which include URL parameters) and set submit values to form values.
+        $this->_formValues = array_merge((array) $this->_formValues, $this->_defaults);
+        $this->_submitValues = $this->_formValues;
+      }
```

Comment:
I made some tests locally with moving the code higher level and realised that there is some inconsistency with particular Search forms. For example Activity and Participants search works fine but some other forms behave differently when using URL parameters and force=1 mode. Some of them don't execute searching and some others do search all results (ignoring search conditions). I guess it may be required to review particular search classes to sort this out.